gitignore-mode.el

```diff
@@ -48,7 +48,7 @@
   (set (make-local-variable 'conf-assignment-sign) nil))
 
 ;;;###autoload
-(dolist (pattern (list "/\\.gitignore\\'"
+(dolist (pattern (list "\\.gitignore.*\\'"
```

Comment:
> I think it'd be fine to add this as an entry, but I don't see the need for the general wildcard regexp.

I might be okay with a wildcard too, depending on how many such non-standard file names are in use, but in any case I would like the commit message to contain a link to some documentation that lists those files.